internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -109,11 +125,15 @@ param (
     [bool]$insecure = $false,
     [string]$collector_version = "",
     [bool]$with_fluentd = $true,
+    [bool]$with_dotnet_tracing = $false,
```

Comment:
Isn't this going to be manageable when more runtimes are added? Anyway, an alternative can be searched when we get to that point.

Reply:
I guess we could just have a switch map. When switch is enabled it downloads the sub-script from the repo artifacts. This way the actual team managing specific language can always auto update the script when needed. Plus version parity stays on if specific release needs something special.